samples/DocumentManagement/app.js

```diff
@@ -44,6 +44,8 @@ var client = new DocumentDBClient( host, { masterKey: masterKey });
 // 5.1                  - Update some properties and replace the document
 // 5.2                  - Use ETag and AccessCondition to only replace document if it has not changed
 // 6. upsertDocument    - Update a document if it exists, else create new document
+// 6.1                  - Insert a document which does not exist
+// 6.2                  - Update a document which exists
```

Comment:
As discussed offline, let us write some of these samples in typescript as well.

Reply:
We'll do this in the object-model-refactoring updates.

---
In reply to: [196175664](https://github.com/Azure/azure-cosmos-js/pull/8#discussion_r196175664) [](ancestors = 196175664)